fe/src/main/java/org/apache/doris/analysis/SelectStmt.java

```diff
@@ -17,6 +17,15 @@
 
 package org.apache.doris.analysis;
 
+import com.google.common.base.Preconditions;
```

Comment:
Reorder your import order in your IDE:

org.apache.doris
com
org
java
javax